pheweb/serve/data_access/db.py

```diff
@@ -263,10 +269,41 @@ def get_variant_annotations(self, id_list):
                 }
             }
         )
+        print('ELASTIC GNOMAD get_variant_annotations hits ' + str(annotation['hits']['total']))
+        print('ELASTIC GNOMAD get_variant_annotations took ' + str(annotation['took']))
         return [ {"id": anno["_source"]["genomes_variantId"],
                   "var_data": { k:v for (k,v) in anno["_source"].items() if re.match("genomes_AF_[A-Z]{3}", k) or k == 'genomes_POPMAX' } }
                  for anno in annotation['hits']['hits'] ]
 
+class TabixGnomadDao(AnnotationDB):
```

Comment:
this class does not implement the gene based query declared in the superclass/interface. Add implementation or define a new interface.